dask/dataframe/tests/test_dataframe.py

```diff
@@ -1821,3 +1821,19 @@ def test_columns_assignment():
     ddf[['a', 'b']] = ddf2[['y', 'z']]
 
     eq(df, ddf)
+
+
+@pytest.mark.parametrize("idx", [
+    np.arange(100),
+    sorted(np.random.random(size=100)),
+    pd.date_range('20150101', periods=100)
+])
```

Comment:
Thoughts on adding an extra parameter for skipna?  If so then we should probably add an NA or two into `pdf`